src/app/pages/fields/governments/government-detail.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Router } from '@angular/router';
+import { Country } from 'app/models/country.model';
+import { CountriesService } from 'app/services/countries.service';
+import { AuthService } from 'app/services/auth.service';
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'otp-government-detail',
+  templateUrl: './government-detail.component.html',
+  styleUrls: ['./government-detail.component.scss']
+})
+export class GovernmentDetailComponent implements OnInit {
+
+  private countries: Country[];
+  private titleText: String = 'New Government';
+
+  constructor(
+    private auth: AuthService,
+    private countriesService: CountriesService,
+    private router: Router
+  ) {
+    this.countries = new Array<Country>();
```

Comment:
You don't need to do that, you can just keep:
```js
private countries: Country[] = [];
```

You have done the same for other components.